### PR TITLE
Added: Current 🇧🇷⏳ Brazilian time in header

### LINKED ISSUE #27: Brazil time in header
Put the current brazil time in the header, similair to how it was on the ohayo-goededagu blog

### PR DESCRIPTION
**Pull request by drikusroor**
Resolves #27

### CODE REVIEW

File: src/blog/templates/blog/includes/brazil-time.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% load brazil_time %}
+
+<span id="brazilTime" class="p-1 rounded text-xs bg-gray-700 text-gray-100">
```
Comment (naomi-eliasar) [inline src/blog/templates/blog/includes/brazil-time.html:3]: I think it will be prettier if the background matches the blue, so bg-blue-800

**Comment by naomi-eliasar** [comment]
There needs to be a mobile layout, now it just wraps it, which looks a bit strange in my opinion

**Comment by naomi-eliasar** [comment]
I think it will be more readable if the date is in normal human order, so day - month - year

File: src/blog/templates/blog/includes/brazil-time.html (hunk 1 above)
Comment (drikusroor) [reply src/blog/templates/blog/includes/brazil-time.html:3]:
```suggestion
<span id="brazilTime" class="p-1 rounded text-xs bg-blue-800 text-gray-100">
```

**Comment by drikusroor** [comment]
> I think it will be more readable if the date is in normal human order, so day - month - year

Normal human order? 😂 https://en.wikipedia.org/wiki/List_of_date_formats_by_country

**Comment by naomi-eliasar** [comment]
> > I think it will be more readable if the date is in normal human order, so day - month - year
> 
> Normal human order? 😂 https://en.wikipedia.org/wiki/List_of_date_formats_by_country

DMY is the right order, the others have it wrong.
The end.

**Pull request merged**